internal/app/config.go

```diff
@@ -241,61 +241,61 @@ func (a *App) envVarNatsOverrides() error {
 // nolint:gocyclo // parameter validation is cyclomatic
 func (a *App) envVarServerserviceOverrides() error {
 	if a.v.GetString("serverservice.endpoint") != "" {
-		a.Config.ServerserviceOptions.Endpoint = a.v.GetString("serverservice.endpoint")
+		a.Config.FleetDBAPIOptions.Endpoint = a.v.GetString("serverservice.endpoint")
 	}
 
-	endpointURL, err := url.Parse(a.Config.ServerserviceOptions.Endpoint)
+	endpointURL, err := url.Parse(a.Config.FleetDBAPIOptions.Endpoint)
 	if err != nil {
 		return errors.New("serverservice endpoint URL error: " + err.Error())
 	}
 
-	a.Config.ServerserviceOptions.EndpointURL = endpointURL
+	a.Config.FleetDBAPIOptions.EndpointURL = endpointURL
 
 	if a.v.GetString("serverservice.disable.oauth") != "" {
-		a.Config.ServerserviceOptions.DisableOAuth = a.v.GetBool("serverservice.disable.oauth")
```

Comment:
should change the names on the environment variables too

Reply:
yep, same as my previous reply